rest-api/test/unit_test/dao_test/biobank_order_dao_test.py

```diff
@@ -15,9 +15,10 @@
 
 
 class BiobankOrderDaoTest(SqlTestBase):
+  config.getSettingList(config.DNA_SAMPLE_TEST_CODES)
   _A_TEST = BIOBANK_TESTS[0]
   _B_TEST = BIOBANK_TESTS[1]
-  _DNA_TEST = BIOBANK_DNA_TESTS[0]
+  _DNA_TEST = config.getSettingList(config.DNA_SAMPLE_TEST_CODES)[0]
```

Comment:
Try just hardcoding this to the first value from the config, and see if it passes.

You can also try inlining the import of config, instead of importing it at the top of the file.

(I believe the problem here is that config is getting imported before memcache is set up for the test.)